### PR TITLE
GOK-181 | Configure GOK drug list to bahmni

### PR DESCRIPTION
**Pull request by parvathy00**
Trello link -> https://trello.com/c/pCAm99m6/181-configure-gok-drug-list-to-bahmni

### CODE REVIEW

File: masterdata/configuration/drugs/drug.csv [hunk 1]
```diff
@@ -515,4 +515,36 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
...
+,,Clotrimazole 400 mg tablet,Clotrimazole,Tablet,400mg,
+,,Dicyclomine Hydrochloride Drops 10mg/5ml 1x10ml,Dicyclomine hydrochloride,Drops,10mg/5ml,
+,,Domperidone Syrup 1mg/ml 1x30ml,Domperidone,Syrup,1mg/ml,
+,,Ferrous sulphate tablet 100 mg iron element & folic acid .5 mg,Ferrous sulfate,dried,Tablet,,
+,,Methyl rosanilium chloride/Aqueous solution, Gentian violet,Solution,.05%,
+,,Salbutamol sulphate Syrup 2mg/5ml 1x100ml,Albuterol sulfate,Syrup,2mg/5ml,
+,,Salbutamol sulphate tablet 2mg,Albuterol sulfate,Tablet,2mg,
+,,Salbutamol sulphate tablet 4mg 1x10x10,Albuterol sulfate,Tablet,4mg,
+,,Theophylline +Etophylline syrup 23+77 mg,Etofylline,Syrup,23+77 mg,
+,,Theophylline +Etophylline tablet 23+77 mg,Theophylline,Tablet,23+77 mg,
```
Comment (gsluthra) [inline masterdata/configuration/drugs/drug.csv]: This isn't likely correct. Its a combination drug, but mapped to a single concept. Should be mapped to a combination concept. Please validate with akhil/angshu.

Comment (parvathy00) [reply masterdata/configuration/drugs/drug.csv]: Sure, will check with them

Comment (parvathy00) [reply masterdata/configuration/drugs/drug.csv]: @gsluthra The combination concept isn't available in CIEL. So created a combination concept (Etophylline / Theophylline) and updated the code.

**Review by deepthi-mantena — APPROVED** [review]

File: masterdata/configuration/concepts/defaultFormConcepts.csv [hunk 2]
```diff
@@ -53,4 +53,6 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Descript
...
 5e67d33c-95fc-4122-ba9b-98a3adf57de0,,,Symptoms of The Reaction,Symptoms of The Reaction,,,,,Misc,Text,,,,,,,,,,,,
-19aac1a6-78ca-4302-a7cc-742d6af1dfa9,,,Allergies,Allergies,,,,,Misc,N/A,,,Types of Allergy;Symptoms of The Reaction;,,,,,,,,,
\ No newline at end of file
+19aac1a6-78ca-4302-a7cc-742d6af1dfa9,,,Allergies,Allergies,,,,,Misc,N/A,,,Types of Allergy;Symptoms of The Reaction;,,,,,,,,,
+3ed880f1-9164-4eb6-8e60-cee1c51606e1,,,Wax dissolving solution,,,,,,Drug,N/A,,,,,,,,,,,
```
Comment (gokultw) [inline masterdata/configuration/concepts/defaultFormConcepts.csv:57]: Dont add drugs related concepts in this CSV. This CSV is only for Forms related concepts.
Create a new CSV file for drugs.

Comment (gokultw) [inline masterdata/configuration/concepts/defaultFormConcepts.csv:57]: Please ensure that this CSV file contains only concepts related to forms, and refrain from including any drug-related concepts. If you need to include drug-related information, kindly create a separate CSV file specifically dedicated to drugs.

File: masterdata/configuration/drugs/drug.csv [hunk 3]
```diff
@@ -515,4 +515,36 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
 ,,Vitamin A 50000 IU,Vitamin A,Oral capsule,50000 IU,
 ,,Vitamin A 100000 IU,Vitamin A,Oral capsule,100000 IU,
 ,,Vitamin A 100000 IU/ml,Vitamin A,Oral solution,100000 IU/ml,
-,,Anti-rabies vaccine,Anti-rabies vaccine,,,
\ No newline at end of file
+,,Anti-rabies vaccine,Anti-rabies vaccine,,,
```
Comment (gokultw) [inline masterdata/configuration/drugs/drug.csv:518]: Should we remove all other drugs from this CSV file and only retain the drug list provided by GOK (Government of Karnataka)?

File: masterdata/configuration/concepts/defaultFormConcepts.csv (hunk 2 above)
Comment (parvathy00) [reply masterdata/configuration/concepts/defaultFormConcepts.csv:57]: Moved drug-related concepts to new CSV file

**Pull request merged**